scripts/extensions/sams/package.json

```diff
@@ -16,7 +16,7 @@
     "@types/react": "16.9.44",
     "@types/redux-logger": "^3.0.8",
     "superdesk-code-style": "1.3.0",
-    "typescript": "3.7.2"
+    "typescript": "3.9.7"
```

Comment:
I think we already have typescript@4 in client-core, we could put it here too

Reply:
The `develop` branch is not currently set to 4. I will update the `sams` branch once `develop` updates

Reply:
ah that's right, it's on a PR, nevermind